src/awx_plugins/interfaces/_temporary_private_api.py

```diff
@@ -4,6 +4,52 @@
 The hope is that it will be refactored into something more standardized.
 """
 
+from collections.abc import Callable
+
+
+class Credential:
+    """Input supplied by the user.
+
+    Satisfies ManagedCredentialType inputs want(s).
+    """
+
+    def __init__(self: 'Credential') -> None:
+        self.inputs: dict[str, bool | str | int | float | None] = {}
```

Comment:
CI failure cause: this line is uncovered —
https://app.codecov.io/github/ansible/awx_plugins.interfaces/commit/bdeaf883f6b52e52e7a836313f8cc4c7552770a7/blob/src/awx_plugins/interfaces/_temporary_private_api.py?flags%5B0%5D=pytest#L17